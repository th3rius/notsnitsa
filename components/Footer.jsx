import React from 'react';
import theme from '../styles/theme';

function Footer() {
  return (
    <div className="footer">
      <img src="/facebook.svg" className="icon" />

      <style jsx>{`
        .footer {
          height: 600px;
          background-color: #424874;
          display: flex;
          flex-direction: row;
          align-items: center;
          padding: 0 32px;
        }

        .subtitle {
          font-family: ${theme.fontFamily.mono};
          color: #fff;
          font-size: 250%;
        }

        .icon {
          fill: white;
          width: 32px;
        }
      `}</style>
    </div>
  );
}

export default Footer;
