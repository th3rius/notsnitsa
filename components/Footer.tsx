function Footer() {
  return (
    <footer className="footer" data-testid="footer">
      <style jsx>{`
        .footer {
          height: 440px; /* FIXME */
          background-color: black;
        }
      `}</style>
    </footer>
  );
}

export default Footer;
