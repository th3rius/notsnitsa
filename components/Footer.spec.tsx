import {render, screen} from "@testing-library/react";

import Footer from "./Footer";

it("should match snapshot", () => {
  render(<Footer />);
  const footer = screen.getByTestId("footer");
  expect(footer).toMatchSnapshot();
});
