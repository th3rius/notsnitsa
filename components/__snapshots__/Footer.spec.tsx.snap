// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<footer
  class="footer"
  data-testid="footer"
/>
`;

exports[`should match snapshot: 
    <footer
      class="footer"
      data-testid="footer"
    />
   1`] = `
<footer
  class="footer"
  data-testid="footer"
/>
`;
